### PR TITLE
fix CSI-5218 uid in vg but nguid on storage

### PR DESCRIPTION
**Pull request by ArbelNathan**
Signed-off-by: ArbelNathan <arbel.nathan@ibm.com>

### CODE REVIEW

**Review by matancarmeli7 — APPROVED** [review]
LGMT

File: controllers/servers/csi/volume_group_server.py [hunk 1]
```diff
@@ -109,7 +111,7 @@ def _get_volume_ids_from_volume_group(self, volumes):
                 lock_request_attribute="volume_group_id")
     def ModifyVolumeGroupMembership(self, request, context):
         secrets = request.secrets
-        utils.validate_delete_volume_group_request(request)
+        utils.validate_modify_volume_group_request(request)
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply controllers/servers/csi/volume_group_server.py:114]: CSI-4880

[the remark this answers is not in the document]
Comment (matancarmeli7) [reply controllers/servers/csi/volume_group_server.py:114]: I think it's the wrong ticket

**Review by erantzabari — APPROVED** [review]

File: controllers/servers/csi/volume_group_server.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply controllers/servers/csi/volume_group_server.py:114]: fixed

**Pull request merged**